class TryCatchCase1
{
	public static void main(String...args) {
		try {
			System.out.println("Hii");
		}
		catch(ArithmeticException e) {
			
		}
	}
}